packages/apollo-server-core/src/plugin/landingPage/default/index.ts

```diff
@@ -140,12 +46,97 @@ function encodeConfig(config: LandingPageConfig): string {
   return JSON.stringify(encodeURIComponent(JSON.stringify(config)));
 }
 
+const getEmbeddedExplorerHTML = (
+  version: string,
+  config: ApolloServerPluginEmbeddedLandingPageProductionDefaultOptions,
+) => {
+  interface EmbeddableExplorerOptions {
+    graphRef: string;
+    target: string;
+
+    initialState?: {
+      document?: string;
+      variables?: Record<string, any>;
+      headers?: Record<string, string>;
+      displayOptions: {
+        docsPanelState?: 'open' | 'closed'; // default to 'open',
+        showHeadersAndEnvVars?: boolean; // default to `false`
+        theme?: 'dark' | 'light';
+      };
+    };
+    persistExplorerState?: boolean; // defaults to 'false'
+
+    endpointUrl: string;
+  }
+  const embeddedExplorerParams: Omit<EmbeddableExplorerOptions, 'endpointUrl'> =
+    {
+      ...config,
+      target: '#embeddableExplorer',
+      initialState: {
+        ...config,
+        displayOptions: {
+          ...config.displayOptions,
+        },
+      },
+    };
+
+  return `
+<div
+style="width: 100vw; height: 100vh;"
+id="embeddableExplorer"
+></div>
+<script src="https://embeddable-explorer.cdn.apollographql.com/${version}/embeddable-explorer.umd.production.min.js"></script>
+<script>
+  var endpointUrl = window.location.href;
+  var embeddedExplorerConfig = ${JSON.stringify(embeddedExplorerParams)};
+  new window.EmbeddedExplorer({
+    ...embeddedExplorerConfig,
+    endpointUrl,
+  });
+</script>
+`;
+};
+
+const getEmbeddedSandboxHTML = () => {
+  // (version: string) => {
+  // TODO (Maya) update this to fetch from the `embeddable-sandbox` repo when the GCS service key is set up correctly
```

Comment:
I am blocked on infrastructure / TF PRs to set up a service account & cli integration to the[ embeddable-sandbox bucket](https://console.cloud.google.com/storage/browser/embeddable-sandbox;tab=permissions?project=mdg-services&prefix=&forceOnObjectsSortingFiltering=false) which should hold the parent page built files. 
For now I uploaded some files to the embeddable-explorer repo under `mayaTest_sandbox` for testing purposes. This needs to be updated before merge, obviously